tracpro/polls/tasks.py

```diff
@@ -92,7 +92,9 @@ def fetch_org_updated_runs(org):
     incomplete_responses = Response.get_update_required(org)
 
     if incomplete_responses:
-        runs = client.get_runs(ids=[r.flow_run_id for r in incomplete_responses])
+        runs = []
+        for i in range(0, len(incomplete_responses), 15):
```

Comment:
That looks like it would ignore everything else and fetch the only first 15. If we are going with itertools then we will have to do something like this. Which is fine
```python
runs = client.get_runs(ids=map(lambda x: x.flow_run_id, incomplete_responses[:len(incomplete_responses)%15))
for chunk in itertools.izip(*(iter(incomplete_responses[len(incomplete_responses)%15:]),) * 15):
    runs += client.get_runs(ids=map(lambda x: x.flow_run_id, chunk)
```